backend/src/layer/logic/image_review/test/image_hoster_mock.rs

```diff
@@ -0,0 +1,54 @@
+use std::sync::{Arc, Mutex};
+
+use async_trait::async_trait;
+
+use crate::interface::image_hoster::{model::ImageMetaData, ImageHoster, ImageHosterError, Result};
+
+pub const PHOTO_ID_TO_FAIL_CHECK_EXISTENCE: &str =
```

Comment:
why do we call these photos now?

Reply:
because of the parameters in check_existence and check_licence. I changed the constants' names. The interfaces need updating

Reply:
i dont think trait parameter names matter (to the compiler), so these could just be updated.